components/Auth/Login.tsx

```diff
@@ -0,0 +1,173 @@
+        if (res.code !== 200) {
+          const error: ErrorOption = {
+            message: res.message.substring(9),
+          }
+          methods.setError('password', error)
+        }
+      })
+      .catch((err) => console.log(err))
+  }
+
+  return (
+    <VStack flex={1} w="100%" justifyContent="space-between" color="black">
+      <Text fontSize={'3xl'} fontWeight="bold">
+        Login
+      </Text>
+      <GoogleSignInComponent />
+      <HStack w="100%">
+        <Divider border="1px solid gray.400" />
+        <Text> or </Text>
+        <Divider border="1px solid gray.400" />
+      </HStack>
+
+      <VStack w="100%" maxW="xs" as="form">
+        <Controller
+          control={methods.control}
+          name="email"
+          render={({ field, fieldState }) => {
+            return (
+              <VStack w="100%">
+                <Input
+                  borderColor={fieldState.error ? 'red' : 'inherit'}
+                  type="email"
+                  value={field.value}
+                  placeholder="Email Address"
+                  onChange={field.onChange}
+                  onBlur={field.onBlur}
+                  onKeyDown={(e) => {
+                    if (e.key == 'Enter') {
+                      methods.setFocus('password')
+                    }
+                  }}
+                />
+                {fieldState.error && (
+                  <Text w="100%" color="red">
+                    {fieldState.error.message}
+                  </Text>
+                )}
+              </VStack>
+            )
+          }}
+        />
+        <Controller
+          control={methods.control}
+          name="password"
+          render={({ field, fieldState }) => {
+            return (
+              <VStack w="100%">
+                <InputGroup>
+                  <Input
+                    ref={field.ref}
+                    onBlur={field.onBlur}
+                    borderColor={fieldState.error ? 'red' : 'inherit'}
+                    type={showPassword ? 'text' : 'password'}
+                    value={field.value}
+                    placeholder="Password"
+                    onChange={field.onChange}
+                    onKeyDown={(e) => {
+                      if (e.key == 'Enter') {
+                        methods.handleSubmit(onSubmit)()
+                      }
+                    }}
+                  />
+                  <InputRightElement
+                    // eslint-disable-next-line react/no-children-prop
+                    children={
+                      <Button
+                        variant="ghost"
+                        backgroundColor="inherit"
+                        px={6}
+                        mr={2}
+                        fontWeight="light"
+                        onClick={() => setShowPassword(!showPassword)}
+                      >
+                        {showPassword ? 'Hide' : 'Show'}
```

Comment:
Check with design on this but imo something like an [eye icon](https://feathericons.com/?query=eye) would be better than `Hide/Show`

Reply:
Totally agree!